etc/docs/utils.ts

```diff
@@ -0,0 +1,95 @@
+import { createInterface } from "readline";
+import yargs from "yargs";
+import { hideBin } from "yargs/helpers";
+
+export interface JsonVersionSchema {
```

Comment:
nit: we have the common pattern of indenting 2 spaces but this file has 4 spaces. Other ts in this PR are also 2.